src/renderer/modules/components/RadioItem.tsx

```diff
@@ -39,6 +38,7 @@ export const Radio = (await waitForModule(filters.bySource(radioStr)).then((mod)
 
 interface RadioItemProps extends RadioProps {
   note?: string;
+  style?: React.CSSProperties;
```

Comment:
I think style can be used on the original component. Did you test that? (applies to other files as well)

Reply:
Yeah I did test. Where I removed the style prop and moved it to the FormItem then the original component doesn't have a style prop.

Reply:
So to confirm, setting the style doesn't change anything?

Reply:
exactly